xilem/src/style.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Traits used to set custom styles on elements.
+
+use masonry_winit::core::Property;
+use masonry_winit::properties::types::Gradient;
+use masonry_winit::properties::{
+    Background, BorderColor, BorderWidth, BoxShadow, CornerRadius, Padding,
+};
+use vello::peniko::Color;
+
+/// Marker trait implement by elements to signal that a given property can be set on them.
```

Comment:
`s/elements/views/`

Being consistent about terminology helps understanding. This applies to this entire PR.